pkg/server/status_test.go

```diff
@@ -35,6 +35,54 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TestDetailsRemoteNode verifies that requesting details for a remote node
+// returns the correct node information.
+func TestDetailsRemoteNode(t *testing.T) {
```

Comment:
The `acceptance/status-server` roachtest today hits a bunch of status server endpoints with different node IDs and expects 200s.

It also does a specific check of the details endpoint to confirm the functionality of nodes returning correct info. IMO that does not need to be in roachtest (and the new `db-console/endpoints` roachtest does not check this functionality for `/details` endpoint specifically). Instead that coverage is added here as a unit test since we want to replace the status server roachtest eventually.

Reply:
nice, appreciate you not completely removing the functionality from the codebase 🙏